modules/cfe_testcase/src/fs_header_test.c

```diff
@@ -105,6 +107,8 @@ void TestTimeStamp(void)
     cFE_FTAssert_VOIDCALL(CFE_FS_InitHeader(&Header, TestDescription, CFE_FS_SubType_ES_ERLOG));
     UtAssert_INT32_EQ(CFE_FS_WriteHeader(fd, &Header), sizeof(CFE_FS_Header_t));
     UtAssert_INT32_EQ(CFE_FS_SetTimestamp(fd, NewTimestamp), CFE_SUCCESS);
+    UtAssert_INT32_EQ(OS_lseek(fd, sizeof(Header.Description), OS_SEEK_CUR), sizeof(CFE_FS_Header_t));
```

Comment:
This could break if the header get's updated... how about comparing the current offset to the location after the time field by using ofsetof the time field in the header + the sizeof the time field?